src/main/java/org/jabref/logic/texparser/DefaultTexParser.java

```diff
@@ -0,0 +1,181 @@
+package org.jabref.logic.texparser;
+
+import java.io.IOException;
+import java.io.LineNumberReader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.jabref.model.database.BibDatabase;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.FieldName;
+import org.jabref.model.texparser.Citation;
+import org.jabref.model.texparser.TexParser;
+import org.jabref.model.texparser.TexParserResult;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DefaultTexParser implements TexParser {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(DefaultTexParser.class);
+
+    /**
+     * It is allowed to add new cite commands for pattern matching.
+     */
+    private static final String[] CITE_COMMANDS = new String[] {"cite", "citep", "citet"};
```

Comment:
There are a few more commands, some of them have a special syntax (like `\parencites[p. 4]{RefA}[p. 5]{RefB}`). See e.g.:
https://github.com/sublimehq/Packages/blob/8c76dd38b2c0049247ec3f57c5b47e07ab52b178/LaTeX/LaTeX.sublime-syntax#L665-L666

Reply:
The [Biblatex-documentation](http://mirror.physik-pool.tu-berlin.de/pub/CTAN/macros/latex/contrib/biblatex/doc/biblatex.pdf) lists them in chapter 3.8 .

Reply:
Thanks! Now it supports many citation commands, even though multicite commands are not ready yet.